.env.example

```
@@ -40,6 +40,9 @@ display_model = epd2in13g_V2
 # Screen rotation - by default we rotate the screen 90 degrees to reach a landscape orientation. If your screen is upside down, try 270.
 screen_rotation = 90
 refresh_interval = 90
+# INFO avoids high-volume driver and HTTP diagnostics on constrained Pi storage.
+# Set to DEBUG temporarily when collecting detailed troubleshooting logs.
+display_log_level = INFO
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The display_log_level  key should be in uppercase

(LowercaseKey)